site/_docs/reference.md

```diff
@@ -2002,14 +2002,14 @@ Note:
 Example Data:
 
 ```JSON
-{"a": [10, true]}
+{"a": [10, true],"b": "[10, true]"}
 ```
 
 Comparison:
```

Comment:
i think here we can remove this line and following lines, and put the result set directly.